testsuite/module-publish/src/packages/simple/sources/Simple.move

```diff
@@ -12,6 +12,7 @@ module 0xABCD::simple {
     use std::signer;
     use std::string::{Self, String, utf8};
     use std::vector;
+    use aptos_framework::aggregator_v2::{Self, Aggregator};
```

Comment:
One think to consider for benchmarks: we put all simple workloads in one module, how does it affect performance? Module loading and fetching all these bytes is not that cheap, I guess. Do we want to keep different workloads separately, or is there a reason all "simple" things are in `Simple.move`?

nit: `Simple.move` --> `simple.move`?